bless/ssh/public_keys/rsa_public_key.py

```diff
@@ -12,6 +12,34 @@
 from cryptography.hazmat.primitives.asymmetric.rsa import RSAPublicNumbers
 
 
+def check_small_primes(n):
+    """
+    Returns True if n is divisible by a number in SMALL_PRIMES.
+    Based on the MPL licensed
+    https://github.com/letsencrypt/boulder/blob/58e27c0964a62772e7864e8a12e565ef8a975035/core/good_key.go
+    """
+    small_primes = [
+        2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47,
+        53, 59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107,
+        109, 113, 127, 131, 137, 139, 149, 151, 157, 163, 167,
+        173, 179, 181, 191, 193, 197, 199, 211, 223, 227, 229,
+        233, 239, 241, 251, 257, 263, 269, 271, 277, 281, 283,
+        293, 307, 311, 313, 317, 331, 337, 347, 349, 353, 359,
+        367, 373, 379, 383, 389, 397, 401, 409, 419, 421, 431,
+        433, 439, 443, 449, 457, 461, 463, 467, 479, 487, 491,
+        499, 503, 509, 521, 523, 541, 547, 557, 563, 569, 571,
+        577, 587, 593, 599, 601, 607, 613, 617, 619, 631, 641,
+        643, 647, 653, 659, 661, 673, 677, 683, 691, 701, 709,
+        719, 727, 733, 739, 743, 751
+    ]
+    for prime in small_primes:
+        if prime >= n:
```

Comment:
What is the intent behind 36-37?  This doesn't seem like the intended behavior of this function.
It looks like line 81 should prevent this check from ever being true.